### PR TITLE
Dry generators + standardize error messages

### PR DESCRIPTION
**Pull request by accessd**

### CODE REVIEW

File: lib/lotus/generators/generatable.rb [hunk 1]
```diff
@@ -46,6 +46,11 @@ def template_source_path
       def target_path
         raise NotImplementedError, "must implement the target_path method"
       end
+
+      def argument_blank?(value)
+        value.nil? || value.strip.empty?
```
Comment (jodosha) [inline lib/lotus/generators/generatable.rb]: @accessd Can you please replace it with `Lotus::Utils::String.new(value).blank?`? This method has a better support for UTF-8 chars. Thank you :+1:

Comment (accessd) [inline lib/lotus/generators/generatable.rb]: @jodosha There's no `blank?` method. Should I add it?

something like:
```
def blank?
  @string.nil? || /\A[[:space:]]*\z/ === @string
end
```

**Comment by accessd** [comment]
rebased

**Comment by accessd** [comment]
How about to add `blank?` to `Hanami::Utils::String`? https://github.com/accessd/utils/commit/ec2928384294c5941c9e429cd7a81362139630b7

File: lib/hanami/generators/generatable.rb [hunk 2]
```diff
@@ -46,6 +46,11 @@ def template_source_path
       def target_path
         raise NotImplementedError, "must implement the target_path method"
       end
+
+      def argument_blank?(value)
+        value.nil? || value.strip.empty?
```
Comment (jodosha) [inline lib/hanami/generators/generatable.rb]: @accessd This doesn't respect UTF-8 chars.

**Comment by accessd** [comment]
updated. using the new method `Hanami::Utils::Blank.blank?`

**Comment by jodosha** [comment]
@accessd Thank you. Just merged this! :+1:

**Pull request merged**